widgets/contact/contact.php

```diff
@@ -236,7 +236,16 @@ function get_widget_form() {
 				'label'  => __( 'Spam Protection', 'so-widgets-bundle' ),
 				'hide'   => true,
 				'fields' => array(
-
+					'honeypot' => array(
+						'type'        => 'checkbox',
+						'label'       => __( 'Honeypot', 'so-widgets-bundle' ),
+						'description' => __( 'Add an invisible field to your form that if filled out, the form will be rejected.', 'so-widgets-bundle' ),
```

Comment:
@AlexGStapleton You ok with the following?

`Adds a hidden form field that only bots can see. The form will reject the submission if the hidden field is populated.`

Reply:
Yes, that's much better.